### PR TITLE
fix #63 all php indentation

### LINKED ISSUE #63: Fix all PHP indentation

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: lib/functions-misc.php [hunk 1]
```diff
@@ -19,23 +19,23 @@ function new_add_post_thumbnail_column($cols){
 }
 add_action('manage_posts_custom_column', 'new_display_post_thumbnail_column', 5, 2);
 function new_display_post_thumbnail_column($col, $id){
-  switch($col){
-    case 'new_post_thumb':
-    if( function_exists('the_post_thumbnail') ) {
+  switch($col) {
+  case 'new_post_thumb':
```
Comment (varffs) [inline lib/functions-misc.php]: shouldnt this be indented and the if too?

Comment (mixcas) [inline lib/functions-misc.php]: idk
why is that even a switch with a single case? :s

Comment (varffs) [inline lib/functions-misc.php]: looks like it should be: http://php.net/manual/en/control-structures.switch.php

Comment (mixcas) [inline lib/functions-misc.php]: again, do we need a switch statement here?

**Comment by varffs** [comment]
@mixcas check my refactor of the switch thing and merge if you are happy

**Pull request merged**